website/static/js/nodeControl.js

```diff
@@ -174,7 +174,11 @@ var ProjectViewModel = function(data, options) {
     };
 
     self.hasIdentifiers = ko.pureComputed(function() {
-        return !!(self.doi() && self.ark());
+        return !!(self.doi());
```

Comment:
`hasIdentifiers()` is also used in `project.mako` where it assumes that both a doi and ark are present.  Seems like this could break that behavior.  Should we instead have a `hasDOI` and `hasARK` and leave `hasIdentifiers` as-is?

Reply:
That's related with my another pr which is on staging3.https://github.com/CenterForOpenScience/osf.io/pull/8113. In that pr, we removed the ability to create new ark but we still want to show all the arks which are already created. Since this pr will definitely go in after that one. When dealing with the front end, I created the conditional situation to show ark based on what we want for the future and we don't need to come back and fix this code when both of them got merged in. I should link that pr. Sry for the confusion.

Reply:
ok thanks @chennan47 !